aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.NSwag/Startup.cs

```diff
@@ -26,8 +26,11 @@ public void Configure(IApplicationBuilder app)
 
             // Register the Swagger generator and the Swagger UI middlewares
             app.UseOpenApi();
-            app.UseSwaggerUi3();
-
+            app.UseOpenApi();
```

Comment:
For these old versions, we need to provide `env` for this to compile.  For example:
public void Configure(IApplicationBuilder app, IHostingEnvironment env)

Reply:
I did

Reply:
Maybe we are not talking about the same thing?  I see:
` public void Configure(IApplicationBuilder app)`

To use `env` it would have to be provided in the following to compile for that version:
public void Configure(IApplicationBuilder app, IHostingEnvironment env)

Reply:
> Maybe we are not talking about the same thing? I see: ` public void Configure(IApplicationBuilder app)`
> 
> To use `env` it would have to be provided in the following to compile for that version: `public void Configure(IApplicationBuilder app, IHostingEnvironment env)`

If you see incorrect code, open an issue and fix. I'm scrambling to cover all `UseSwaggerUi` calls in dev mode.